### DISCUSSION TITLE
Add a toggleable feature for unused assets

### DISCUSSION THREAD

**Discussion by shps18ppt** [ppy/osu#24617] [Ideas] [answered]
I am submitting a feature request regarding the inclusion of toggleable background images and the optimization of unused assets in osu! skins, specifically in the lazer build of the game.

One example of an unused asset is the background image for ranking panels. Currently, the background image in the ranking panel does not show in lazer, resulting in a less visually appealing experience for players. Additionally, there are other assets in skins from stable that are not fully utilized in lazer, leading to wastage of storage space.

I propose the implementation of a toggleable feature in lazer that allows users to enable or disable background images for various elements, including backgrounds in ranking panels, mod icons, etc. in skins. This will provide players with the option to enjoy the aesthetic value and immersive experience offered by background images. By toggling this feature, players can choose between a streamlined interface or a more visually appealing one.

Furthermore, it would be beneficial to optimize the usage of all assets in skins to ensure that they are fully utilized. This will prevent unnecessary storage consumption and ensure that the effort put into creating and downloading skins is fully appreciated.

I believe that incorporating these features will significantly enhance the customization options available in osu! while optimizing resource allocation. It will contribute to a more enjoyable and immersive gameplay experience for all players.

Thank you for considering this feature request. I appreciate your continued dedication to improving the lazer build and look forward to the potential implementation of these features in future updates.

Best regards.

**Comment by peppy** [comment, marked as answer]
Is this written by an AI?

Anyway, implementing skin support is an ongoing effort and will eventually be realised.
